src/okta/fetchBatchOfApplicationUsers.ts

```diff
@@ -0,0 +1,249 @@
+import {
+  IntegrationError,
+  IntegrationInstanceAuthorizationError,
+} from '@jupiterone/jupiter-managed-integration-sdk';
+
+import {
+  OktaExecutionContext,
+  OktaIntegrationStepIterationState,
+} from '../types';
+import extractCursorFromNextUri from '../util/extractCursorFromNextUri';
+import logIfForbiddenOrNotFound from '../util/logIfForbidden';
+import retryApiCall from '../util/retryApiCall';
+import {
+  OktaApplicationCacheEntry,
+  OktaApplicationUser,
+  OktaApplicationUserCacheEntry,
+  OktaCacheState,
+  OktaQueryParams,
+} from './types';
+
+export default async function fetchBatchOfApplicationUsers(
```

Comment:
I think we should remove this code from this PR because it is designed to support iterating calls to a single Lambda function. The new `iterateApplicationUsers` we aim for will be far, far simpler.

Reply:
Removed